Umbraco Cloud Release notes March (#5064)
umbraco/UmbracoDocs: umbraco-cloud/release-notes/2023-03-releasenotes.md

# Release Notes, February 2023

## Key Takeaways

* **Umbraco Baseline Support For All Cloud Regions** - Umbraco Cloud now offers baseline support for all three regions - West Europe, East US, and South UK.
* **Simplified Access to Project Team Page** - As an administrator for a Cloud organization you will be able to access the project team for all the cloud projects of the organization.
* **Monthly Bandwidth Usage** - You find an update to the “Usage” page where you now can see the total bandwidth used for the past seven months.
* **Upgrade Project Flow** - The previous Upgrade plan feature has been migrated into a new, improved Upgrade Project flow, where you can change the project plan and change to a dedicated resource in one go!

## Umbraco Baseline Support For All Cloud Regions

> Umbraco Cloud has recently expanded its **baseline feature to** include the **East US and South UK regions**, providing users with a standardized set of features across all regions.

Reviewer: ```suggestion
We've expanded the **baseline feature to** include the **East US and South UK regions**. This provides users with a standardized set of features across all regions.
```

> If you have had experience working with Umbraco Cloud baselines, you will know that the basic idea is that you have a project that contains all your standard Umbraco packages/components, maybe configured with some default Document Types, which you want to use as baseline for future projects. When you've made changes to your Baseline project, you can then push these changes out to all the child projects with a click of a button.

Reviewer: ```suggestion
If you have had experience working with Umbraco Cloud baselines, you will know that the basic idea about baselines. You have a project that contains your standard Umbraco packages/components, configured with default Document Types, which can be used as baseline for future projects. When you've made changes to your Baseline project, you can push these changes out to the child projects with a click of a button.
```

> If you primary have been working with Umbraco Cloud in the US or UK regions and accordingly are not that familiar with the Umbraco Cloud baseline feature you can read more about it at [Umbraco Docs](https://docs.umbraco.com/umbraco-cloud/getting-started/baselines), or see an introduction at the [Umbraco Learning Base](https://www.youtube.com/watch?v=Ci1Hm-bH98Y) on how to work with baseline in Umbraco Cloud.

Reviewer: ```suggestion
If you primarily have been working with Umbraco Cloud in the US or UK regions and accordingly are not that familiar with the Umbraco Cloud baseline feature you can read more about it at [Umbraco Docs](https://docs.umbraco.com/umbraco-cloud/getting-started/baselines), or see an introduction at the [Umbraco Learning Base](https://www.youtube.com/watch?v=Ci1Hm-bH98Y) on how to work with baseline in Umbraco Cloud.
```

## Simplified Access to Project Team Page

As an administrator of your organization in Umbraco Cloud, you now have access to the **project team page** (called the "Edit Team") of all your **Umbraco cloud projects in your organization**.

> The project team page can easily be accessed from the overview showing all the organization's projects. The administrator can by clicking the icon listed to the right of the individual project., where he will be able to invite users to join the project team with a set of project permissions and backoffice user group memberships or revoke access to the project of an existing project team member.

Reviewer: ```suggestion
The project team page can easily be accessed from the overview showing all the organization's projects. The administrator can by clicking the icon listed to the right of the individual project. He will be able to invite users to the project team with a set of permissions and backoffice user group memberships. He can also revoke access to the project of an existing project team member.
```

![OrgAdminEditUserPage](images/OrgAdminEditUserPage.gif)

> This change provides a centralized and convenient way for the organization administrator to manage the team members' access to the cloud projects, ensuring that trusted individuals of your organization are able to grant and restrict access to cloud projects in Umbraco Cloud.

Reviewer: ```suggestion
This change provides a centralized and convenient way for the organization administrator to manage the team members' access to the cloud projects. Hereby Ensuring that trusted individuals of your organization are able to grant and restrict access to cloud projects in Umbraco Cloud.
```

The new addition overall improves security, enhances collaboration, and streamlines the workflow for the organization's cloud projects.

## Monthly Bandwidth Usage

> We have launched a small update on the usage page that allows you to see the **total bandwidth used every month for the past seven months**.

Reviewer: ```suggestion
We've launched a minor update on the usage page. The update lets you see the **total bandwidth used every month for the past seven months**.
```
